.travis.yml

```diff
@@ -36,6 +36,7 @@ env:
 
 install:
   - "pip install -r requirements.txt"
+  - "pip install -r requirements/test-local.txt"
```

Comment:
Is this still needed if we can invoke the fixtures programmatically? Does even just installing pytest-django cause problems?